pkg/web/buildsHandler.go

```diff
@@ -1,21 +1,58 @@
 package web
 
 import (
+	"fmt"
 	"net/http"
 
 	"github.com/aerogear/mobile-developer-console/pkg/mobile"
 	"github.com/labstack/echo"
+	buildV1 "github.com/openshift/api/build/v1"
 )
 
 type MobileBuildsHandler struct {
-	namespace   string
-	buildsCRUDL mobile.BuildCRUDL
+	buildsCRUDL        mobile.BuildCRUDL
+	namespace          string
+	openshiftMasterURL string
 }
 
-func NewMobileBuildsHandler(buildsCRUDL mobile.BuildCRUDL, namespace string) *MobileBuildsHandler {
+type extendedBuildList struct {
+	buildV1.BuildList
+	Items []extendedBuild `json:"items" protobuf:"bytes,2,rep,name=items"`
+}
+
+type extendedBuild struct {
+	buildV1.Build
+	BuildURL string `json:"buildUrl,inline"`
+}
+
+func NewMobileBuildsHandler(buildsCRUDL mobile.BuildCRUDL, namespace string, openshiftMasterURL string) *MobileBuildsHandler {
 	return &MobileBuildsHandler{
-		buildsCRUDL: buildsCRUDL,
-		namespace:   namespace,
+		buildsCRUDL:        buildsCRUDL,
+		namespace:          namespace,
+		openshiftMasterURL: openshiftMasterURL,
+	}
+}
+
+// Extend existing struct buildV1.BuildList so each build contains URL
+// pointing to build in OpenShift Console
+func extendBuildList(bl buildV1.BuildList, mbh MobileBuildsHandler) *extendedBuildList {
```

Comment:
don't think we need to pass the `MobileBuildsHandler` object here. It's better to just pass the url and the namespace value as args explicitly here.